src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/AddAzureVpnClientRevokedCertificateCommand.cs

```diff
@@ -59,10 +59,6 @@ public override void Execute()
         {
 
             base.Execute();
-            if (!this.IsVirtualNetworkGatewayPresent(ResourceGroupName, VirtualNetworkGatewayName))
```

Comment:
Can you explain what the intention of the PR is?  It looks like you are just removing existence checks - won't this cause the cmdlet to fail with a less useful message later?

Also please ensure that there is a test that covers the bad error message that you are trying to change